plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -369,7 +369,12 @@ public function verify_cot_data( $args = array(), $assoc_args = array() ) {
 			return WP_CLI::success(
 				sprintf(
 					/* Translators: %1$d is the number of migrated orders and %2$f is time taken */
-					_n( '%1$d order was verified, in %2$f seconds', '%1$d orders were verified in %2$f seconds', $processed, 'woocommerce' ),
+					_n(
+						'%1$d order was verified in %2$f seconds',
```

Comment:
We probably want to end these sentences with a dot ".", for consistency with the other messages below.